docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ---------------------------- |
+| For Edge airgap clusters, manifests attached to packs are not applied during cluster deployment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Add the manifest as a layer directly instead of attaching it to a pack. For more information, refer to [Add a Manifest](../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-manifest-addon.md).                                                                                                                                                                                                                                                                                                                                                                                    | Nov 15, 2024       | Edge                         |
```

Comment:
```suggestion
| For Edge airgap clusters, manifests attached to packs are not applied during cluster deployment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Add the manifest as a layer directly instead of attaching it to a pack. For more information, refer to [Add a Manifest](../profiles/cluster-profiles/create-cluster-profiles/create-addon-profile/create-manifest-addon.md).                                                                                                                                                                                                                                                                                                                                                                                    | November 15, 2024       | Edge                         |
```